src/autoscaler/eventgenerator/generator/evaluator_test.go

```diff
@@ -467,10 +527,66 @@ var _ = Describe("Evaluator", func() {
 					})
 				})
 			})
+
+			Context("multiple triggers", func() {
+				BeforeEach(func() {
+					scalingEngine.RouteToHandler("POST", urlPath, ghttp.RespondWith(http.StatusOK, "successful"))
+					Expect(triggerChan).To(BeSent(triggerArrayMultipleTriggers))
+				})
+				Context("when only the first trigger breaches", func() {
+					BeforeEach(func() {
+						database.RetrieveAppMetricsStub = func(appId string, metricType string, start int64, end int64) ([]*models.AppMetric, error) {
+							return appMetricGEBreach, nil
+						}
+					})
+					It("should send trigger alarm to scaling engine", func() {
+						Eventually(scalingEngine.ReceivedRequests).Should(HaveLen(1))
+						Eventually(logger.LogMessages).Should(ContainElement(ContainSubstring("send trigger alarm to scaling engine")))
+					})
+				})
+
+				Context("when only second tigger breaches", func() {
+					BeforeEach(func() {
+						database.RetrieveAppMetricsStub = func(appId string, metricType string, start int64, end int64) ([]*models.AppMetric, error) {
+							return appMetricLEBreach, nil
+						}
+					})
+					It("should send trigger alarm to scaling engine", func() {
+						Eventually(scalingEngine.ReceivedRequests).Should(HaveLen(1))
+						Eventually(logger.LogMessages).Should(ContainElement(ContainSubstring("send trigger alarm to scaling engine")))
+					})
+				})
+
+				Context("when both tiggers breach", func() {
+					BeforeEach(func() {
+						database.RetrieveAppMetricsStub = func(appId string, metricType string, start int64, end int64) ([]*models.AppMetric, error) {
+							return appMetricMultipleTriggerAllBreach, nil
+						}
+					})
+					It("should send one trigger alarm to scaling engine", func() {
+						Eventually(scalingEngine.ReceivedRequests).Should(HaveLen(1))
+						Eventually(logger.LogMessages).Should(ContainElement(ContainSubstring("send trigger alarm to scaling engine")))
+					})
+				})
+
+				Context("when no tigger breaches", func() {
+					BeforeEach(func() {
+						database.RetrieveAppMetricsStub = func(appId string, metricType string, start int64, end int64) ([]*models.AppMetric, error) {
+							return appMetricMultipleTriggerAllNotBreach, nil
+						}
+					})
+					It("should not send trigger alarm to scaling engine", func() {
+						Eventually(scalingEngine.ReceivedRequests).Should(HaveLen(0))
+						Eventually(logger.LogMessages).Should(ContainElement(ContainSubstring("should not send trigger alarm to scaling engine")))
```

Comment:
yes, you are right. 

I will remove this one once we agreed the strategy on how to handle the trigger conflict.